frontend/src/metabase/query_builder/components/VisualizationError.jsx

```diff
@@ -90,13 +91,12 @@ class VisualizationError extends Component {
     via: PropTypes.object.isRequired,
     question: PropTypes.object.isRequired,
     duration: PropTypes.number.isRequired,
-    error: PropTypes.object.isRequired,
+    error: PropTypes.any.isRequired,
```

Comment:
It's expected as both an object and a string

Reply:
nit

```
PropTypes.oneOfType([
  PropTypes.object,
  PropTypes.string
]),
```